snowplow/iglu-client-embedded/schemas/com.metabase/setup/jsonschema/1-0-2

```diff
@@ -0,0 +1,104 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Setup flow",
+  "self": {
+    "vendor": "com.metabase",
+    "name": "setup",
+    "format": "jsonschema",
+    "version": "1-0-2"
+  },
+  "type": "object",
+  "properties": {
+    "event": {
+      "description": "Event name",
+      "type": "string",
+      "enum": [
+        "step_seen",
+        "usage_reason_selected",
+        "database_selected",
+        "add_data_later_clicked"
+      ],
+      "maxLength": 1024
+    },
+    "version": {
+      "description": "String describing the version of onboarding we're on",
+      "type": "string",
+      "maxLength": 1024
+    },
+    "step": {
+      "description": "String describing the step that the set up step",
+      "type": [
+        "string",
+        "null"
+      ],
+      "enum": [
+        "welcome",
+        "language",
+        "user_info",
+        "db_connection",
+        "usage_question",
+        "db_scheduling",
+        "data_usage",
+        "completed"
+      ],
+      "maxLength": 1024
+    },
+    "step_number": {
+      "description": "Integer describing the order of the referenced step",
+      "type": [
+        "integer",
+        "null"
+      ],
+      "enum": [
```

Comment:
What if we made the maximum of 6 instead of an enum here?

Reply:
Ah, didn't even see that lol. (I think it's going to fail at the last step as it is not)
I'm not sure why we even have that, could it be that it's for optimization purposes?
As we'd release this tomorrow, I'd follow the Chesterton's Fence rule and just add "6" to the enum, we can figure a proper solution when we'll add the license activation step